specs/windows/chassis_info.table

```diff
@@ -0,0 +1,21 @@
+table_name("chassis_info")
+description("Display information pertaining to the chassis and its security status.")
+schema([
```

Comment:
Try to keep sentence case for all the descriptions for consistency

Reply:
Not sure I follow this - do you want the descriptions to be sentences or just short descriptions? I seem to notice there isn't a consistency between some of the existing tables so it would be good to know which one we want

Reply:
Sorry for being unclear, I just meant with regards to capitalization

Reply:
Aha oops, sku is lowercase. Got that one.

Reply:
If there's any others I missed lemme know